src/pollable_output_stream.rs

```diff
@@ -155,11 +155,11 @@ pub struct OutputStreamAsyncWrite<T: IsA<PollableOutputStream>>(
 );
 
 impl<T: IsA<PollableOutputStream>> OutputStreamAsyncWrite<T> {
-    pub fn into_input_stream(self) -> T {
+    pub fn into_output_stream(self) -> T {
         self.0
     }
 
-    pub fn input_stream(&self) -> &T {
+    pub fn output_stream(&self) -> &T {
```

Comment:
This is an API change, and I'm aware of code out there using the old (broken) names. If anything we can deprecate the broken names here and add the new ones for this bugfix release (and that probably makes sense to do!).

Reply:
Something like https://github.com/gtk-rs/gio/pull/279

Reply:
Updated PR